### PR TITLE
UI overhaul

### PR DESCRIPTION
**Pull request by iwangy**
added colored coded rules, made home page look better, tidied things up

### CODE REVIEW

**Comment by abhaybd** [comment]
Looks like it's not up to date with master, can you pull the latest commit and merge it in? Conflicts should be pretty minimal I think

File: frontend/src/JoinForm.js [hunk 1]
```diff
@@ -49,20 +49,18 @@ class JoinForm extends React.Component {
...
                     <tr>
                         <td>
-                            Name:
-                        </td>
-                        <td>
-                            <input type="text" value={this.state.name} size='10' onChange={this.nameChange}/>
+                            <input type="text" value={this.state.name} size='10' placeholder = "Altaïr" maxlength = '12' onChange={this.nameChange}/>
                         </td>
                     </tr>
+                    <div id = "create-code">Enter Acess Code
```
Comment (abhaybd) [inline frontend/src/JoinForm.js]: Minor typo, can you fix that?

Comment (iwangy) [reply frontend/src/JoinForm.js]: what's the typo there?

Comment (abhaybd) [reply frontend/src/JoinForm.js]: Acess should be Access

**Comment by abhaybd** [comment]
Looks good, i'll squash n merge

**Pull request merged**